libtiledbsoma/src/external/include/thread_pool/status.h

```diff
@@ -59,7 +59,7 @@
 
 #define tdb_delete_array(p) \
   if (p) {                  \
-    delete[] p;             \
+    free((void *)p);        \
```

Comment:
super naive question: given that this is all used in C++ code, wouldn't an alternative fix be to use `new` in the `tbd_new_array` macro?  I.e., use new/delete, rather than malloc/free?

Reply:
@bkmartinjr either one is fine as long is it matches:

* `malloc` and `free`
* `new` and `delete`
* `new[]` and `delete []`

Please let me know your preference.

Reply:
I looked at Core, and it uses new/delete, not malloc/delete.

Reply:
Will do, thanks!

Reply:
For the record these will necessarily be `new []` and `delete []`